.github/ISSUE_TEMPLATE/blank_issue.md

```diff
@@ -0,0 +1,8 @@
+---
+name: Blank Issue
```

Comment:
I see that there's a default option on the bottom for blank issues too but happy to leave this as is!:

![image](https://user-images.githubusercontent.com/5422235/148619218-8808fef3-4af1-4e37-b4dd-b1403acd85be.png)

Reply:
It's a bit hard to see, which is why I added it.